### DISCUSSION TITLE
GitHub Workflows

### DISCUSSION THREAD

**Discussion by Dekker500** [hbstack/.github#8] [Ideas]
I was interested in your framework, and followed the installation instructions to clone, npm ci, npm run dev. 

I was very surprised when during the FIRST RUN, I got notifications from GitHub regarding Cloudflare Pages and GitHub Pages - My first thought was that my blank demo site had been published to GitHub and Cloudflare!!! Not knowing what was published increased my anxiety. The commands to ci and run shouldn't mean deploy! If the above was the result of the git push, then the following is even more important, as just the act of pushing it created a financial burden, as minutes were billed.

I strongly suggest that your default workflow is LOCAL, without hosting the site in GitHub. The commands to ci and run shouldn't mean deploy! If the above was the result of the git push, then the following is even more important, as just the act of pushing it created a financial burden, as minutes were billed.

Include instructions to MANUALLY ACTIVATE the workflows for 
- Hosting on using GitHub to store our site
- Deploying to GitHub pages
- Deploying to Cloudflare pages
- etc.

Also, suggest you add comments/help text to those workflow pages to know how to customize it to our own site...

**Comment by razonyang** [comment]
Sorry about that, I didn't think about *private* repos, the initial purpose is to make the workflow out-of-box.

> If the above was the result of the git push.

Yep, those workflows are triggered when pushing a commit to repo, make senses to remove those workflows from the template.

Now, those workflows were removed from starter template, the deployment instructions can be found on https://hb.hugomods.com/en/docs/deployment/.

Thanks for pointing this out, and sorry for the inconvenience.
